tests/sandbox/sandbox.sh

```diff
@@ -15,6 +15,7 @@ echo "Setting up Python environment..."
 if [ ! -f "${VENV_DIR}/bin/activate" ]; then
     python3.8 -m venv "${VENV_DIR}"
 fi
+# shellcheck source=/dev/null
```

Comment:
Is this the "source only from hardcoded paths" warning? If so we can turn it off globally I think.

Reply:
Yeah, I think turning it off makes sense.